.env.test

```diff
@@ -1,9 +1,25 @@
 # GitHub
 FORCE_USER_TOKEN_GITHUB_CLIENT="false"
 GH_USER_TOKEN="ghp_BLAHBLAHBLAH"
-GH_APP_IDENTIFIER="1234"
 GH_WEBHOOK_SECRET="webhooksecret"
-GH_APP_SECRET_KEY="top \nsecret\n key"
+
+# some functionality is org-agnostic
+GH_APP_1_ORG_SLUG="Enterprise"
+GH_APP_1_IDENTIFIER="1234"
+GH_APP_1_SECRET_KEY="top \nsecret\n key"
+GH_APP_1_ISSUES_PROJECT_NODE_ID="PVT_blahblah"
+GH_APP_1_PRODUCT_AREA_FIELD_ID="PVTSSF_flooflah"
+GH_APP_1_STATUS_FIELD_ID="PVTSSF_zimzimzim"
+GH_APP_1_RESPONSE_DUE_DATE_FIELD_ID="PVTF_zamzamzam"
+
+# other functionality is hardwired for getsentry
+GH_APP_2_ORG_SLUG="getsentry"
+GH_APP_2_IDENTIFIER="7890"
+GH_APP_2_SECRET_KEY="super \nsecret\n key"
+GH_APP_2_ISSUES_PROJECT_NODE_ID="PVT_flahflah"
+GH_APP_2_PRODUCT_AREA_FIELD_ID="PVTSSF_moomah"
+GH_APP_2_STATUS_FIELD_ID="PVTSSF_mizmizmi"
+GH_APP_2_RESPONSE_DUE_DATE_FIELD_ID="PVTF_mazmaza"
```

Comment:
This is nice because it gives us multi-org testing, we're kind forced into it.

Reply:
good call

Reply:
Done in d86f107e8517a7b06fafda6522dba1c4d375f63f, also updates `.env.example`.